phpunit.xml

```diff
@@ -32,11 +32,12 @@
     <php>
         <env name="APP_ENV" value="testing"/>
         <env name="CACHE_DRIVER" value="array"/>
+        <env name="DB_DATABASE" value="rogue_test"/>
+        <env name="DS_ENABLE_BLINK" value="false"/>
+        <env name="DS_ENABLE_GLIDE" value="true"/>
+        <env name="SCOUT_DRIVER" value="null"/>
```

Comment:
When running tests locally and then checking the Algolia `campaigns` index, I was initially surprised to see that the number of records jumped significantly! Realized that running the tests was causing creating and updates to the index. So setting the value in PHPUnit to "null" keeps the tests from sending data to Algolia!